tests/app_tiovx_linux_capture_encode_test.c

```diff
@@ -73,14 +73,20 @@
 #define INPUT_WIDTH  (1920)
 #define INPUT_HEIGHT (1080)
 
+#define OUTPUT_WIDTH  (1280)
+#define OUTPUT_HEIGHT (768)
+
 #define SENSOR_NAME "SENSOR_SONY_IMX219_RPI"
```

Comment:
@sinha-shreyash Can you please add the details on, why you had to add MSC in the pipeline